### DISCUSSION TITLE
jellyfin low power encoding

### DISCUSSION THREAD

**Discussion by chenks** [community-scripts/ProxmoxVE#1002] [Q&A]
i'm running jellyfin using the script from here (Unprivileged ubuntu LXC).
all is working well and transcoding (when required) works as it should (with iGPU passthru)

the PVE host is a 12th gen i3-12100

over on the jellyfin forum it was suggested that i can also enable low-power encoding to give faster OpenCL tone-mapping transcoding.

the instructions the provide show how to install, but it's a bit vague as to whether the required drivers/config should be done on the pve host or inside the LXC container.

has anyone done this and/or advise?

https://jellyfin.org/docs/general/administration/hardware-acceleration/intel/#low-power-encoding

**Comment by chenks** [comment]
anyone?

**Reply by jsapede** [reply]
what about cloning your lxc and have a test one for installing inside lxc ?

**Reply by chenks** [reply]
that doesn't really answer the question i posed.

**Reply by Mellowlynx** [reply]
"over on the jellyfin forum it was suggested"
I think this question can better be posted there, as the person suggested it, knows more that users here.

I also agree with [jsapede](https://github.com/jsapede). Having a home lab and apps running, also mains you are willing to thinker and learn by trying and error and using a search engine.

What might help you:
As the LXC is running the software, (and communicating with the hardware (because passthrough)) here is where you edit configs or add drivers.
On the host itself, you should not need to make any changes.

**Reply by chenks** [reply]
> "over on the jellyfin forum it was suggested"
> I think this question can better be posted there, as the person suggested it, knows more that users here.

the person that suggested it was a jellyfin dev, and they said they weren't sure if it needed installed on the host or inside the LXC - hence asking here as jellyfin was installed using the script from here.

> I also agree with [jsapede](https://github.com/jsapede). Having a home lab and apps running, also mains you are willing to thinker and learn by trying and error and using a search engine.

forgive me for having the galll to use the Q&A section of the discussions section that is specifically in place for this kind of thing!

> As the LXC is running the software, (and communicating with the hardware (because passthrough)) here is where you edit configs or add drivers.
> On the host itself, you should not need to make any changes.

hence asking the question here!
